docs/_articles/en/faq.md

```diff
@@ -37,7 +37,7 @@ If the language is not already supported, please log an Issue as per
 ["How do I get support for additional languages?"](./en/faq/#q-how-do-i-get-support-for-additional-languages).
 
 #### Q: How do I add new rules for Language X?
```

Comment:
Would it be worth mentioning somewhere that rulesets can be added as standalone XML? We can also add that rulesets are deprecated to match with PMD's decision for v7?